### PR TITLE
input component done

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**

### CODE REVIEW

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+.input-group{
+    width: 80%;
+    height: auto; ;
+    padding: 25px;
+}
+
+.input--text{
+    padding: 10px;
```
Comment (AshutoshM29) [inline input/input.css]: Trying using rem, just to keep the code uniform

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Nice work!!
You can merge the branch

File: input/input.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply input/input.css]: done

**Comment by Sahil25061999** [comment]
looks good no changes required

**Comment by kotesh-arya** [comment]
Looking good enough for a merge

**Review by kotesh-arya — APPROVED** [review]
It looks perfect for me 👍🏻

**Comment by git-0r** [comment]
in documentation input font color is black, hard to see on dark background.

**Pull request merged**